Userland/Libraries/LibWeb/HTML/HTMLLinkElement.cpp

```diff
@@ -294,8 +294,48 @@ void HTMLLinkElement::default_fetch_and_process_linked_resource()
                     // 2. If either of the following conditions are met:
                     // - bodyBytes is null or failure; or
                     // - response's status is not an ok status,
+                    if (body_bytes.template has<Empty>()) {
+                        // CORS cross-origin responses in the No CORS request mode provide an opaque filtered response, which is the original response
+                        // with certain attributes removed/changed.
+
+                        // The relevant effect it has is setting the body to `null`, which means `body_bytes` has `Empty` here. This effectively
+                        // disables cross-origin linked resources (e.g. stylesheets).
+
+                        // However, the web actually depends on this, especially for stylesheets retrieved from a cross-origin CDN. For example,
+                        // Shopify websites request stylesheets from `cdn.shopify.com` and Substack websites request stylesheets from `substackcdn.com`.
+
+                        // This makes this a specification bug, as this code was written from it.
```

Comment:
Feels like we should be good bois and report the spec bug to WHATWG, or link one if there's already an existing issue.

Reply:
Done: https://github.com/whatwg/html/issues/9066

Reply:
Also added to the comment